lib/utils/report.js

```diff
@@ -107,7 +109,11 @@ module.exports = function (violation) {
 		warningProperties.word = word;
 	}
 
-	const warningMessage = _.get(result.stylelint, ['customMessages', ruleName], message);
+	if (!result.stylelint.customMessages) {
+		result.stylelint.customMessages = {};
+	}
+
+	const warningMessage = result.stylelint.customMessages[ruleName] || message;
```

Comment:
[sugggestion] Same here:

```suggestion
	const warningMessage = (result.stylelint.customMessages && result.stylelint.customMessages[ruleName]) || message;
```